setup.py

```diff
@@ -73,8 +73,8 @@ def find_version(*file_paths):
 if python_version[0] < 3:
     print("Only Python > 3.5 is supported")
     sys.exit(0)
-elif  python_version[1] < 5:
-    print("Only Python > 3.5 is supported")
+elif  python_version[1] < 8:
+    print("Only Python > 3.8 is supported")
```

Comment:
This is probably asking the user too much. Let's not explicitly prohibit 3.5-3.7 here.

Reply:
Should these lines be deleted from script?